akka-sample-cluster-scala/src/main/protobuf/TransformationMessages.proto

```diff
@@ -0,0 +1,18 @@
+syntax = "proto3";
+
+package sample.cluster.transformation;
```

Comment:
the actual package name becomes `sample.cluster.transformation.TransformationMessages`, which is a strange name for a package.

Try
```
option java_package = "sample.cluster.transformation";
```

instead of above `package`.

Two more options that I'm not completely sure what they mean:
```
option java_multiple_files = true;
option java_outer_classname = "TransformationMessages";
```